core/lib/benches/format-routing.rs

```diff
@@ -13,7 +13,7 @@ fn post() -> &'static str { "post" }
 
 fn rocket() -> rocket::Rocket {
     let config = Config::new(Environment::Production).unwrap();
-    rocket::custom(config, false).mount("/", routes![get, post])
+    rocket::custom(config).mount("/", routes![get, post])
```

Comment:
I believe this will enable logging, where it wasn't before. This was changed in 965c90af; something like`config.log = Off` may be appropriate here.

Reply:
Addressed.